email_strides/ce.py

```diff
@@ -0,0 +1,72 @@
+import logging
+
+import boto3
+from botocore.config import Config as BotoConfig
+
+LOG = logging.getLogger(__name__)
+LOG.setLevel(logging.DEBUG)
+
+cost_metric = "NetAmortizedCost"
+
+# Use adaptive mode in an attempt to optimize retry back-off
+ce_config = BotoConfig(
+    retries={
+        "mode": "adaptive",  # default mode is legacy
+    }
+)
+ce_client = boto3.client("ce", config=ce_config)
+
+
+def get_ce_service_costs(period):
```

Comment:
by "different information" I mean that these CE queries are different than the other reports, and parsing the response requires knowing the length of the GroupBy value.